pages/api-reference/connect/exchanges/create-exchange-for-an-account.mdx

```diff
@@ -0,0 +1,419 @@
+---
+layout: guides
+product: connect
+title: "Create an exchange for an account"
+guide:
+  step: 3
+meta:
+  title: "Create an exchange for an account | Dwolla API Documentation"
+  description: Create an exchange for a reseller partner or client account.
+---
+
+# Create an exchange for an account
+
+This section contains information on how to create an exchange for a reseller partner or client account. The creation of an exchange serves as the “hand-shake” between Dwolla and a trusted ecosystem partner. The creation of an exchange requires a `_link` to the exchange partner and tokenized data which encapsulates limited permissioned access between an ecosystem partner and Dwolla.
+
+### HTTP request
+
+> `POST https://api.dwolla.com/exchanges`
+
+### Request parameters
+
+| Parameter | Required    | Type   | Description                                                                                                                                                                                                   |
+| --------- | ----------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| \_links   | yes         | object | A \_links JSON object containing an exchange partner link. See example raw request and response below.                                                                                                        |
+| token     | conditional | string | A third party provider processor token. **Required** for MX and Flinks integrations.                                                                                                                          |
+| finicity  | conditional | object | Finicity resource object containing access key **receipt** information. **Required** for Finicity integration. See `finicity` JSON object below (including nested `product` and `accessPeriod` JSON objects). |
+
+#### Finicity JSON Object
```

Comment:
Just adding for future consideration/iteration, looks like this would be a good candidate for nested tables.